src/tests/test_example_diffs.rs

```diff
@@ -1608,6 +1608,36 @@ src/align.rs:71: impl<'a> Alignment<'a> { │
         assert!(output.contains(r"https://invent.kde.org/utilities/konsole/-/commit/94907c0f136f46dc46ffae2dc92dca9af7eb7c2e"));
     }
 
+    #[test]
+    fn test_hyperlinks_commit_link_oneline_log() {
+        let config = integration_test_utils::make_config_from_args(&[
+            // Use commit-style to ensure test to pass in Github Actions
+            "--commit-style",
+            "blue",
+            "--hyperlinks",
+            "--hyperlinks-commit-link-format",
+            "https://invent.kde.org/utilities/konsole/-/commit/{commit}",
+        ]);
+        let output = integration_test_utils::run_delta(GIT_LOG_ONELINE, &config);
+        println!("{}", output);
```

Comment:
(These print statements can be deleted once the tests are passing with standard output redirected)

Reply:
Oh yeah, good pickup! I'll remove them once the Github build is sorted out.